Domain/LoginDomain/Sources/UseCases/Login/LoginUseCase.swift

```diff
@@ -23,19 +24,19 @@ public final class LoginUseCase: LoginUseCaseInterface {
 		return loginRepository.fetchEmailLoginAPI(email: email, password: password)
 	}
 	
-	public func fetchEmailConfirm(email: String) -> Single<EmailConfirmResponse> {
+	public func fetchEmailConfirm(email: String) -> Single<EmptyResponse> {
 		return loginRepository.fetchEmailConfirmAPI(email: email)
 	}
```

Comment:
```suggestion
public func fetchEmailConfirm(email: String) -> Single<EmptyResponse> {
  loginRepository.fetchEmailConfirmAPI(email: email)
}
```

이거는 참고차 알아두면, 단순 return 구문만 있는 경우, `return` 키워드를 생략할 수 있습니다.

동일한 예시로 `LoginRepository`의 코드가 되겠네용
```
public func fetchEmailConfirmAPI(email: String) -> Single<EmptyResponse> {
  request(endPoint: .emailConfirm(email: email))
}

```

전체적으로 통일성 있게 수정해도 좋고, 아니면 개념적인 부분으로 알고 있으면 좋을 것 같아요

수정하지 않는다면 읽고 resolve 해주세용